android/app/src/main/java/org/fossasia/openevent/api/processor/SpeakerListResponseProcessor.java

```diff
@@ -31,11 +31,13 @@ public void onResponse(Call<List<Speaker>> call, final Response<List<Speaker>> r
                 ArrayList<String> queries = new ArrayList<>();
 
                 for (Speaker speaker : response.body()) {
+                    if(speaker.getSession()!=null){
```

Comment:
Can you please fix the indentation?

Also, appropriate spacing will be great
`if (speaker.getSession() != null) {`

Try to write code like it will be when you reformat using IDE :smile: 

Otherwise looks good